src/NetAnalyzers/Core/AnalyzerReleases.Unshipped.md

```diff
@@ -1 +1,5 @@
 ; Please do not edit this file manually, it should only be updated through code fix application.
+### New Rules
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+CA1011 | Design | Hidden | ConsiderPassingBaseTypesAsParameters, [Documentation](https://docs.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca1011)
```

Comment:
New analyzers added to NetAnalyzers project should not be targeting main branch, but a specific .NET release branch (`release/7.0.1xx` I believe): https://github.com/dotnet/roslyn-analyzers/blob/main/docs/NetCore_GettingStarted.md#branch-definitions

@buyaa-n @carlossanlop - this PR is porting an old FxCop CA rule to .NET Analyzers. Does this need to go through dotnet/runtime triage?